app/models/image.rb

```diff
@@ -0,0 +1,15 @@
+class Image < ActiveRecord::Base
+
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+  belongs_to :user
+
+  has_attached_file :image, :styles => { :large => "600x600>", :medium => "300x300>", :thumb => "100x100>" }, :default_url => "/images/:style/missing.png"
+  validates_attachment_content_type :image, :content_type => /\Aimage\/.*\Z/
+
+  validates :image, :presence => true
+
+  def liked_by? current_user, image
```

Comment:
Unused method argument - `current_user`. If it's necessary, use `_` or `_current_user` as an argument name to indicate that it won't be used. You can also write as `liked_by?(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `image`. If it's necessary, use `_` or `_image` as an argument name to indicate that it won't be used. You can also write as `liked_by?(*)` if you want the method to accept any arguments but don't care about them.